### PR TITLE
CNV-20148: VM export RN

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.12

Issue: [CNV-20148](https://issues.redhat.com//browse/CNV-20148)

Link to docs preview: [Release notes (new features)](https://54125--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-12-release-notes.html#virt-4-12-new)

QE review: @jpeimer 
- [x] QE has approved this change.

Additional information: Links to be added after docs PR is merged.

### CODE REVIEW

**Review by kelbrown20 — COMMENTED** [review]
/lgtm

**Review by jpeimer — CHANGES_REQUESTED** [review]

**Review by kgoldbla — APPROVED** [review]
lgtm

File: virt/virt-4-12-release-notes.adoc [hunk 1]
```diff
@@ -47,8 +47,10 @@ The SVVP Certification applies to:
 //CNV-18488 SVVP for 4.12
 
 //CNV-21611-2
-{VirtProductName} no longer uses the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo. {VirtProductName} is now represented by the image:virt-icon.svg[{VirtProductName},40,40] logo for versions 4.9 and later.
+* {VirtProductName} no longer uses the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo. {VirtProductName} is now represented by the image:virt-icon.svg[{VirtProductName},40,40] logo for versions 4.9 and later.
 
+//CNV-20148 VM export - links to be added after doc PR is merged.
+* You can export and download a volume from a virtual machine (VM), a VM snapshot, or a persistent volume claim (PVC) for forensic analysis by using the `virtctl vmexport` command or by creating a `VirtualMachineExport` custom resource.
```
[the remark this answers is not in the document]
Comment (awels) [reply virt/virt-4-12-release-notes.adoc:53]: Yeah one of the main use cases is exporting the volume to another cluster (or some external storage). The forensic analysis IMO is a secondary use case. I like the recommended wording

[the remark this answers is not in the document]
Comment (apinnick) [reply virt/virt-4-12-release-notes.adoc:53]: Great. I'll add that.

File: virt/virt-4-12-release-notes.adoc [hunk 2]
```diff
@@ -47,8 +47,10 @@ The SVVP Certification applies to:
 //CNV-18488 SVVP for 4.12
 
 //CNV-21611-2
-{VirtProductName} no longer uses the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo. {VirtProductName} is now represented by the image:virt-icon.svg[{VirtProductName},40,40] logo for versions 4.9 and later.
+* {VirtProductName} no longer uses the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo. {VirtProductName} is now represented by the image:virt-icon.svg[{VirtProductName},40,40] logo for versions 4.9 and later.
 
+//CNV-20148 VM export - links to be added after doc PR is merged.
```
[the remark this answers is not in the document]
Comment (apinnick) [reply virt/virt-4-12-release-notes.adoc:52]: Yes, it's merged. Thanks for the reminder. I forgot to add the link. Will fix.

**Pull request merged**